services/src/main/java/org/apache/druid/cli/CliHistorical.java

```diff
@@ -101,6 +101,8 @@ protected List<? extends Module> getModules()
         new JoinableFactoryModule(),
         new HistoricalServiceModule(),
         binder -> {
+          CliCoordinator.validateCentralizedDatasourceSchemaConfig(getProperties());
```

Comment:
Is this validation needed here?

Reply:
In case of incompatible config, doesn't it make sense to fail all the services?

Reply:
No, only the services which actually use that config should even be aware of it.